README.md

```diff
@@ -24,3 +24,32 @@ The connector is created automatically when the application is started. See the
 for a list of all available properties.
 
 The connector can be configured using a configuration file. See `localdev` for an example configuration file.
+
+# Restore-CLI
+
+To restore records from a backup run the program 
+
+```bash
+
+./gradlew :cli:run restore --bucket <s3-backup-bucket> --pattern <topicPattern>
+
+```
+
+The above command restores all records for a given topic to the same topic name.
+
+### All options:
+| Option name        | Short option | Required                                    | Format              | Description                                                                                                                      |
+|--------------------|--------------|---------------------------------------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| bucket             | b            | always                                      | String              | Bucket in which the backup is stored                                                                                             |
```

Comment:
does `--bucket` requires to be the bucket-name or does it also support to specify the path to filter on?

Reply:
It is for bucket only. Other filtering happens via the topic. I noticed now that this assumes backups are only stored `topics/<topicname>/...` at the top level. Is this right?